components/actionbar/metadata/actionbar.yml

```diff
@@ -17,16 +17,16 @@ examples:
                 <use xlink:href="#spectrum-css-icon-DashSmall" />
               </svg>
             </span>
-            <span class="spectrum-Checkbox-label">2 Selected</span>
+            <span class="spectrum-Checkbox-label">2 selected</span>
           </label>
-          <div class="spectrum-ButtonGroup">
-            <button class="spectrum-ActionButton spectrum-ActionButton--quiet">
+          <div class="spectrum-ActionGroup">
+            <button class="spectrum-ActionButton spectrum-ActionButton--quiet spectrum-ActionGroup-item">
```

Comment:
@jianliao could you add a migration guide section and include the need to use `spectrum-ActionGroup-item` and maybe mention `spectrum-Icon--sizeM` as well

Reply:
Done.